### DISCUSSION TITLE
Allow multiple circuits when using multiple trains

### DISCUSSION THREAD

**Discussion by astraylife1** [OpenRCT2/OpenRCT2#22058] [Feature requests]
The game’s default behavior removes the multiple circuit option for powered launch mode as well as continuous circuit mode when there is more than one train on the track. It would be nice if this option was made available when using the cheat “unlock operating limits.” This would be useful for hacked rides, triple launch coasters with multiple trains, etc.

**Comment by autosysops** [comment]
It's probably removed as it would be a dangerous operating mode. I think if this was implemented it should only happen if the cheat to allow for all operating modes is enabled.

**Comment by astraylife1** [comment]
I’m also not sure if the game would even be able to keep track of number of circuits per train or if it would just count any pass through the station.
